doc/docs/stdlib.html

```diff
@@ -360,6 +376,94 @@ <h4 id="manifestJsonEx">std.manifestJsonEx(value, indent)</h4>
 </code>
 
 
+<h4 id="manifestYamlDoc">std.manifestYamlDoc(value)</h4>
+
+<p>Convert the given value to a YAML form.  Note that <code>std.manifestJson</code> could also be
+used for this purpose, but this function will produce more canonical-looking YAML.</p>
+
+<code>std.manifestYamlDoc(
+{
+    x: [1, 2, 3, true, false, null, "string\nstring\n"],
+    y: { a: 1, b: 2, c: [1, 2] },
+})
+</code>
+
+<p>Yields a string containing this YAML:</p>
+
+<code>"x": 
+  - 1
+  - 2
+  - 3
+  - true
+  - false
+  - null
+  - |
+    string
+    string
+"y": 
+  "a": 1
+  "b": 2
+  "c": 
+    - 1
+    - 2
+</code>
+
+
+<h4 id="manifestYamlStream">std.manifestYamlStream(value)</h4>
+
+<p>Given an array of values, emit a YAML "stream", which is a sequence of documents separated by
+<code>---</code>.</p>
+
+<code>std.manifestYamlStream(['a', 1, []])</code>
+
+<p>Yields this string:</p>
+
+<code>---
+"a"
+---
+1
+---
+[]
+...
+</code>
+
+
+<h4 id="manifestXml">std.manifestXml(xml)</h4>
+
+<p>Convert the given <a href="http://www.jsonml.org/">JsonML</a>-encoded value to a string
+containing the XML.</p>
+
+<code>std.manifestXml([
+  'svg',
+  { height: 100, width: 100 },
+  ['circle', { cx: 50, cy: 50, r: 40, stroke: 'black', 'stroke-width': 3, fill: 'red' }],
+])</code>
+
+<p>Yields a string containing this XML (all on one line):</p>
+
+<code>&lt;svg height="100" width="100"&gt;
+  &lt;circle cx="50" cy="50" fill="red" r="40" stroke="black" stroke-width="3"&gt;&lt;/circle&gt;
+&lt;/svg&gt;
+</code>
+
+<p>Which looks like this:</p>
```

Comment:
Nit: technically it says that the string above looks like the circle below, while it only represents that circle.